libs/jsonforms-components/src/lib/Controls/ObjectArray/DeleteDialog.tsx

```diff
@@ -24,6 +24,17 @@ export const DeleteDialog = React.memo(function DeleteDialog({
   title,
   message,
 }: DeleteDialogProps) {
+  useEffect(() => {
+    if (open) {
+      const deleteBtn = document.querySelector('[data-testId="object-array-confirm-button"]') as HTMLButtonElement;
```

Comment:
are you sure the data test id will not conflict with other controls because not having a unique value might cause issues with Howard's tests nightly test